api/src/tests/invalid_post_request_test.rs

```diff
@@ -0,0 +1,205 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::tests::{assert_json, new_test_context};
+
+use serde_json::{json, Value};
+
+#[tokio::test]
+async fn test_invalid_type_argument() {
+    let mut req = signing_message_request();
+    req["payload"]["type_arguments"][0] = json!("Invalid");
+
+    assert_error_msg(req, "Request body deserialize error: parse Move type \"Invalid\" failed, caused by error: unexpected token Name(\"Invalid\"), expected type tag at line 1 column 432").await;
```

Comment:
nit: Do we need these error assertions to be so exact? I worry the tests will now be too brittle and require copy changes even tho behavior and functionality remained the same.

Reply:
Probably not. I just started something simple, and this is the only way to know what exactly looks like when the error happened.

Test running with different debugging env variable outputs differently, so I will need to use partial match instead of exact match